Apps/phgitlab/gitlab_connector.py

```diff
@@ -0,0 +1,404 @@
+
+    def _handle_list_users(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # make rest call
+        ret_val, response = self._make_rest_call('/users', action_result, params=None, headers=self._rest_auth_header)
+
+        if (phantom.is_fail(ret_val)):
+            return action_result.get_status()
+
+        # Add the response into the data section
+        action_result.add_data(response)
+
+        return action_result.set_status(phantom.APP_SUCCESS, "Successfully returned users")
+
+    def _handle_list_projects(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # make rest call
+        ret_val, response = self._make_rest_call('/projects', action_result, params=None, headers=self._rest_auth_header)
+
+        if (phantom.is_fail(ret_val)):
+            return action_result.get_status()
+
+        # Add the response into the data section
+        action_result.add_data(response)
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_list_branches(self, param):
+
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        project_id = param['project_id']
+
+        # make rest call
+        ret_val, response = self._make_rest_call('/projects/{}/repository/branches'.format(project_id), action_result, params=None, headers=self._rest_auth_header)
+
+        if (phantom.is_fail(ret_val)):
+            return action_result.get_status()
+
+        # Add the response into the data section
+        action_result.add_data(response)
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_create_trigger(self, param):
+
+        # Implement the handler here
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        project_id = param['project_id']
+        trigger_name = param['trigger_name']
+        trigger_data = {'description': trigger_name}
+
+        # make rest call
+        ret_val, response = self._make_rest_call('/projects/{}/triggers'.format(project_id), action_result, method="post", params=None, headers=self._rest_auth_header, data=trigger_data)
+
+        if (phantom.is_fail(ret_val)):
+            return action_result.get_status()
+
+        action_result.add_data(response)
+
+        token_dict = {'tigger_token': response['token']}
```

Comment:
Should this be `trigger_token`?

Reply:
Yes, good catch! I made the fix in [this commit](https://github.com/peterbertel/phantom-apps/commit/eb678462e42b5ba6339f1c3a7886438c7fd10cb4).